praetorian_cli/sdk/entities/definitions.py

```
@@ -29,23 +29,51 @@ def add(self, local_filepath, definition_name=None):
             definition_name = os.path.basename(local_filepath)
         return self.api.files.add(local_filepath, f'definitions/{definition_name}')
 
-    def get(self, definition_name, download_directory=os.getcwd()):
+    def get(self, definition_name, download_directory=os.getcwd(), global_=False):
```

Ruff (0.12.2)

Do not perform function call `os.getcwd` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)